### DISCUSSION TITLE
fds initialization

### DISCUSSION THREAD

**Discussion by kjhxjm** [firemodels/fds#12757] [Q&A]
Hello everyone, we are currently working on a project about an indoor fire simulation platform.
Since FDS needs to be modified dynamically, our approach is to perform a simulation first and then end the simulation at a certain moment. Then initialize the results of this simulation into a new simulation. In the new simulation, we can add and delete items at will to achieve the purpose of modification.
I can only use the fdsreader tool to initialize gas, temperature, and heat release rate. My question is how to initialize the combustion of the last simulated object. I don't know where a bed burned to when I ended the last simulation and how it initializes into the next simulation.
Can you give me some ideas and suggestions? Thank you all!

**Comment by mcgratta** [comment]
Search the FDS User's Guide for the word `RESTART`. FDS has a way to stop and start simulations.

**Reply by kjhxjm** [reply]
Thank you very much for your suggestion, but if we restart the simulation, we cannot modify the fds file. Because our system may need to make some modifications to the fds file, such as fire extinguishing, adding or deleting equipment.

**Comment by mcgratta** [comment]
FDS does not have the capability of changing geometry and devices mid-way through a simulation.

**Reply by drjfloyd** [reply]
If you know the types of changes you might make you can build all those OBST with DEVC or CTRL_ID. So if you have an OBST you might need to remove you would have a DEVC or CTRL that had the OBST present during the first simulaton and in the restart you change the state to have the OBST go away,.
